Robust.Shared/Network/Messages/MsgEntity.cs

```diff
@@ -31,7 +31,7 @@ public class MsgEntity : NetMessage
         public uint Sequence { get; set; }
         public GameTick SourceTick { get; set; }
 
-        public override void ReadFromBuffer(NetIncomingMessage buffer)
+        public override unsafe void ReadFromBuffer(NetIncomingMessage buffer)
```

Comment:
```suggestion
        public override void ReadFromBuffer(NetIncomingMessage buffer)
```

Unsafe code was removed so...